data/presets/natural/tree.json

```diff
@@ -1,14 +1,24 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
         "leaf_type_singular",
         "leaf_cycle_singular",
-        "denotation",
-        "height",
-        "diameter"
+        "start_date"
     ],
     "moreFields": [
+        "protected",
+        "age",
+        "sex",
+        "operator",
```

Comment:
0.19% (or is 0.43% the right number?) – so super low…

https://taginfo.openstreetmap.org/tags/?key=natural&value=tree#combinations

Reply:
Ok let add the operator later 👍